docs/gke-installation.md

```diff
@@ -0,0 +1,127 @@
+# Deploying Antrea on a GKE cluster
+
+We support running Antrea inside of GKE clusters on Ubuntu host. Antrea would operate
+in noEncap mode, in which no encapsulation is required for any kind of traffic (Intra Node,
+Inter Node etc) and NetworkPolicies are enforced using OVS.
+
+## GKE Prerequisites
+
+1. Install the Google Cloud SDK (gcloud). Refer to this (https://cloud.google.com/sdk/install)
+
+    ```bash
+    curl https://sdk.cloud.google.com | bash
+    ```
+
+2. Make sure you are authenticated to use the Google Cloud API
+
+    ```bash
+    export ADMIN_USER=user@email.com
+    gcloud auth login
+    ```
+
+3. Create a project or use an existing one
+
+    ```bash
+    export GKE_PROJECT=gke-clusters
+    gcloud projects create $GKE_PROJECT
+    ```
+
+## Creating the cluster
+
+You can use any method to create a GKE cluster (gcloud SDK, gcloud Console etc). The example
+given here is using the Google Cloud SDK.
+
+**Note:** Antrea is supported on Ubuntu Nodes only for GKE cluster. Also, it is must to select service
```

Comment:
In my understanding, it should be "it is a must" but not "it is must", but I trust you will figure out the right grammar.

Reply:
Done